app/services/processors/code_climate_update_all_projects_service.rb

```diff
@@ -1,5 +1,5 @@
-module CodeClimate
-  class UpdateAllProjectsService < BaseService
+module Processors
+  class CodeClimateUpdateAllProjectsService < BaseService
```

Comment:
WDYT about creating a folder for CodeClimate services? And then CodeClimate::UpdateAllProjects ?

Reply:
It's already there https://github.com/rootstrap/rs-code-review-metrics/tree/feature/add-code-climate-update-job-cron/app/services/code_climate
but since all the entry points to `Processors` services were in this folder I moved this class here too. I can move it back though

Reply:
Ohh sorry! My bad! Its ok!

Reply:
The thumb would be to leave in `processors` or to move it back to `code_climate` 🤔 ?